src/client/pages/VergunningDetail/BlauweZoneBedrijven.tsx

```diff
@@ -0,0 +1,34 @@
+import { defaultDateFormat } from '../../../universal/helpers';
+import InfoDetail, {
+  InfoDetailGroup,
+} from '../../components/InfoDetail/InfoDetail';
+import { BZB } from '../../../server/services/vergunningen';
+
+export function BlauweZoneBedrijven({ vergunning }: { vergunning: BZB }) {
```

Comment:
Wat mij betreft gebruik je voor de bestandsnaam het zaaktype, net als de andere vergunningen. In dit geval `BZB.tsx`